src/cljs/hello_world/util.cljs

```diff
@@ -119,12 +125,14 @@
         "reset-button"
         (fn []
           (println "Reset clicked"))
-        this))))
+        this)))
+  (hide-reset-button!))
```

Comment:
Why do we need to hide-reset-button here? :)

Reply:
we make reset button when game start, but we don't need it until the player starts playing the game. We hide it and show it when player clicks (starts) playing puzzle game.

Reply:
So true! 👍